packages/discord.js/src/structures/Message.js

```diff
@@ -417,6 +417,30 @@ class Message extends Base {
     } else {
       this.poll ??= null;
     }
+
+    /**
+     * The call associated with the message
+     * @typedef {Object} MessageCall
+     * @property {?Readonly<Date>} endedAt The time the call ended
+     * @property {?number} endedTimestamp The timestamp the call ended
+     * @property {Snowflake[]} participants The ids of the users that participated in the call
+     */
+
+    if (data.call) {
+      /**
+       * The call associated with the message
+       * @type {?MessageCall}
+       */
+      this.call = {
+        endedTimestamp: data.call.ended_timestamp && Date.parse(data.call.ended_timestamp),
```

Comment:
According to the docs, this might also missing

```suggestion
        endedTimestamp: data.call.ended_timestamp ? Date.parse(data.call.ended_timestamp) : null,
```